assets/constructor_macOS_arm64.patch

```diff
@@ -1,12 +0,0 @@
-diff -Naur constructor-orig/osxpkg.py constructor-patched/osxpkg.py
---- constructor-orig/osxpkg.py	2022-06-02 23:30:08.000000000 +0200
-+++ constructor-patched/osxpkg-patched.py	2022-06-02 23:31:41.000000000 +0200
-@@ -156,6 +156,8 @@
-         options.set('customize', 'never')
-         options.set('customLocation', '/Applications/MNE-Python')
-         options.set('rootVolumeOnly', 'true')
-+        # Be explicit and declare we're only able to work on Apple Silicon
-+        options.set('hostArchitectures', 'arm64')
```

Comment:
... and then also set this depending on the constructor type -- `arm64` for the ARM installer and `x86_64` for the Intel installer.

Reply:
Yes, this one is not present right now, but for some reason it works ok so far? But maybe all those users had Rosetta enabled already.

Reply:
... but this stuff does seem relevant/useful, no?

Reply:
Okay!